package.json

```diff
@@ -25,8 +25,8 @@
   "version": "0.27.1-rc2",
   "private": true,
   "dependencies": {
-    "@hathor/wallet-lib": "1.0.2",
-    "@ledgerhq/hw-transport-node-hid": "6.27.1",
+    "@hathor/wallet-lib": "1.0.4",
+    "@ledgerhq/hw-transport-node-hid": "^6.28",
```

Comment:
question: Could we use a pinned version for this dependency? This has been the default since #422

Reply:
Done